code/drasil-example/Drasil/GlassBR/DataDefs.hs

```diff
@@ -79,8 +78,8 @@ hFromt = dd hFromtQD [makeCite astm2009] Nothing "minThick" [hMin]
 
 --DD3-- (#749)
 
-loadDFEq :: Expr 
-loadDFEq = (sy loadDur / 60) $^ (sy sflawParamM / 16)
+loadDFEq :: Expr
+loadDFEq = (sy loadDur $/ dbl 60) $^ (sy sflawParamM $/ dbl 16)
```

Comment:
I'm pretty sure this 60 comes from a minute or second computation, so is actually exact. No idea about the 16,

Reply:
Thanks, fixed the 60. I also opted to change the 16 to an exact as well since it's an equation. I think it just makes the output look nicer anyways. It's a reasonably low number too.

Reply:
Yes, the 60 is 60 seconds in a minute.  I also don't know the meaning of the 16, but I agree that it is exact.  The source of these GlassBR equations is:

https://github.com/smiths/caseStudies/blob/master/CaseStudies/glass/refs/1BasisForASTME1300AnnealedGlassThicknessSelectionCharts.pdf

There is another paper cited there (Beason and Morgan) that likely explains the 16 better, but I don't think it is worth our time to fret over this, especially when an exact 16 is so likely to be our conclusion.